actors/builtin/miner/miner_actor.go

```diff
@@ -980,7 +990,9 @@ func (a Actor) OnDeferredCronEvent(rt Runtime, payload *CronEventPayload) *adt.E
 	case CronEventProvingPeriod:
 		handleProvingPeriod(rt)
 	case CronEventPreCommitExpiry:
-		checkPrecommitExpiry(rt, payload.Sectors)
+		if payload.Sectors != nil {
```

Comment:
Does this improve correctness?  It looks like it is just an optimization and if so it's not clear its worth doing.

Reply:
Accessing it if nil is definitely wrong. I don't have reason to believe it will be, but this is just on the side of worthwhile defensiveness for me.